documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -1,45 +1,193 @@
-- Interface support: the interface should support limited liquid democracy for delegate selection and approval voting for public goods council candidates. The interface or explorer should display past retroactive PGF winners and past/current continuous funding recipients. Proposal submission for continuous and retroactive funding will happen separately, in whatever manner the public goods council deems fit.
+### Constructing the council
+All valid PGF councils will be established multisignature account addresses. These must be created by the intdended parties that wish to create a council. The council will therefore have the discretion to decide what threshold will be required for their multisig (i.e the "k" in the "k out of n").
+
+The council will be resonsible to publish this address to voters and express their desired `spending_cap`. 
+
+A council consisting of the same members should also be able to propose multiple spending caps (with the same multisig address). These will be voted on as separate councils and votes counted separately.
+
+
+### Voting on the council
+Once the council has been constructed, it can be voted on by governance particpants through a vote as structured below:
+
+```rust
+struct OnChainVote {
+    id: u64,
+    voter: Address,
+    yay: proposalVote,
+}
+```
+
+In turn the proposal vote will include the a structure like:
+
+```rust
+HashSet<(Address, u64)>
+```
+
+The structure contains all the counsils voted, where each cousil is specific as a pair `Address` (the enstablished address of the multisig account) and `u64` (spending cap)
+
+These votes will then be used in order to vote for various PGF councils. Multiple councils can be voted on through a vector as represented above.
+
+
+### CLI
+
+TODOs:
+
+- PGF Votes
+- Construction of PGF council multisigs 
+
+### Dealing with ties
+In the rare occurance of a tie, the council with the lower spending_cap will win the tiebreak.
+
+In the case of equal tiebreaks, the addresses with lower alphabetical order will be chosen. This is very arbitrary due to the expected low frequency.
+
+### Ensuring enough votes 
+
+In order for a new PGF council to be elected (and hence halting the previous council's power), $\frac{1}{3}$ of validating power must vote on the "New PGF Council Proposal". Once this condition has been met, majority vote will be decided as to which PGF council will be elected.
+
+### Electing the council
+
+Once the elected council has been decided upon, the established address corresponding to the multisig is added to the `PGF` internal address, and the `spending_cap` variable is stored.
```

Comment:
we need to track spending_cap, the pgf VP should check that some spending_amount variable (reset to 0 on election of a new council) is incremented by the amount of any transfer

Reply:
we still need to describe this, I think?